src/licenses/LicenseFilter.h

```diff
@@ -0,0 +1,30 @@
+#ifndef TURTLEBROWSER_LICENSEFILTER_H
+#define TURTLEBROWSER_LICENSEFILTER_H
+
+#include "licenses/LicenseCategory.h"
+
+#include <QtCore/QSortFilterProxyModel>
+
+namespace licenses {
```

Comment:
you can consider also a parent namespace `turtlebrowser` for all the classes

Reply:
Hm, yes... that might make sense

Reply:
Added a turtle_browser namespace (with an underscore so I don't get a spelling error 🤣 )

Reply:
deciding a namespacing strategy is important.

I long ago (after being burned various ways) decided that always typing the namespace is the way to go (ie I always type `std::string` never just `string`) so I'm a big fan namespaces but _short_ namespaces!